packages/block-library/src/post-comments-form/editor.scss

```diff
@@ -0,0 +1,25 @@
+.wp-block-post-comments-form {
+	textarea {
+		padding: calc(0.667em + 2px);
+		border: 1px solid $gray-600;
+		font-size: 1em;
+		font-family: inherit;
+	}
+
+	.comment-form {
+		textarea,
+		label {
+			display: block;
+			margin-bottom: 0.25em;
+		}
+	}
+
+	input[name="submit"] {
+		border: none;
+		box-shadow: none;
+		cursor: pointer;
+		display: inline-block;
+		text-align: center;
+		overflow-wrap: break-word;
+	}
+}
```

Comment:
Is it possible that the styling from this file isn't actually being applied? (Try e.g. `color: red` for the `input[name="submit"]`)

I think we might be missing an `@import` line for this file around here: https://github.com/WordPress/gutenberg/blob/102d36681439b7d0ea40d1cd41aa68d0f4adf3f8/packages/block-library/src/editor.scss#L52

(and possibly an `editorStyle` in `block.json`, akin to [this](https://github.com/WordPress/gutenberg/pull/40484/files#diff-189adb33acee7fd799b7bbf136bd14c2499b6363618ea1d1b2c86a96989c7acdR44)?)

Reply:
Yes! I'm not sure how I missed that. In fact I ll remove the custom styling because it seems that it was not even necessary in the first place and only add the:

```css
.wp-block-post-comments-form * {
  pointer-events: none;
}
```

Reply:
d2d7ccc2bb3e0a2bdc8b9b3018dccd2ea6d3ea2a